bee-message/tests/parents.rs

```diff
@@ -30,6 +31,22 @@ fn new_invalid_more_than_max() {
     ));
 }
 
+#[test]
+fn new_invalid_not_sorted() {
+    let mut inner = rand_message_ids(8);
+    inner.reverse();
+
+    assert!(matches!(Parents::new(inner.clone()), Err(Error::ParentsNotUniqueSorted)));
+}
+
+#[test]
+fn new_invalid_not_unique() {
+    let mut inner = rand_message_ids(7);
+    inner.push(inner[0]);
```

Comment:
It's not straightforward that it tests unicity. Pushing the first element to the end also isn't sorted. Probably better to have two consecutive same elements ?

Reply:
I've updated to repeat the last element instead, so all IDs will be sorted, just with one repeated element.